### PR TITLE
Cerberus observer

### PR DESCRIPTION
**Pull request by paigerube14**
Want to be able to monitor different components during chaos and perf scaling using cerberus as an observer pod

https://docs.ci.openshift.org/docs/internals/observer-pods/

### CODE REVIEW

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

File: ci-operator/step-registry/redhat-chaos/cerberus/redhat-chaos-cerberus-commands.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+while [ ! -f "${KUBECONFIG}" ]; do
+  printf "%s: waiting for %s\n" "$(date --utc --iso=s)" "${KUBECONFIG}"
+  sleep 10
+done
+printf "%s: acquired %s\n" "$(date --utc --iso=s)" "${KUBECONFIG}"
+
+ls
+
+git clone https://github.com/redhat-chaos/krkn-hub.git
```
[the remark this answers is not in the document]
Comment (paigerube14) [reply ci-operator/step-registry/redhat-chaos/cerberus/redhat-chaos-cerberus-commands.sh:16]: This is a separate repo we need to run this observer, but thank you for the hint with the from: src. I think that's what I was missing to get the certain installations in my container to run

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

File: ci-operator/step-registry/redhat-chaos/cerberus/redhat-chaos-cerberus-commands.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (bbguimaraes) [reply ci-operator/step-registry/redhat-chaos/cerberus/redhat-chaos-cerberus-commands.sh:16]: You can also use https://docs.ci.openshift.org/docs/getting-started/examples#publishing-an-image-for-reuse if you don't need the absolute latest version of the repository.

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

File: ci-operator/step-registry/redhat-chaos/cerberus/OWNERS [hunk 2]
```diff
@@ -0,0 +1,7 @@
+approvers:
+  - chaitanyaenr
+  - yogananth-subramanian
+  - paigerube14
+  - tsebastiani
+reviewers:
```
Comment (chaitanyaenr) [inline ci-operator/step-registry/redhat-chaos/cerberus/OWNERS:6]: @paigerube14 we might want to add you and me to the reviewers list as well. Thoughts?

Comment (paigerube14) [reply ci-operator/step-registry/redhat-chaos/cerberus/OWNERS:6]: I think approvers auto get "reviewer" access as well. I'll read up on it to be sure though

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-4.14-nightly-krkn-hub-tests-414

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-4.14-nightly-krkn-hub-tests-414

**Comment by paigerube14** [comment]
Waiting to use image published from: https://github.com/openshift/release/pull/43114

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-4.14-nightly-krkn-hub-tests-414

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-4.14-nightly-krkn-hub-tests-414

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-4.14-nightly-krkn-hub-tests-414

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-4.14-nightly-krkn-hub-tests-414

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-4.14-nightly-krkn-hub-tests-414

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-4.14-nightly-krkn-hub-tests-414

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-4.14-nightly-krkn-hub-tests-414

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-4.14-nightly-krkn-hub-tests-414

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-4.14-nightly-krkn-hub-tests-414

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-4.14-nightly-krkn-hub-tests-414

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-4.14-nightly-krkn-hub-tests-414

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-4.14-nightly-krkn-hub-tests-414

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-4.14-nightly-krkn-hub-tests-414

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-4.14-nightly-krkn-hub-tests-414

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-4.14-nightly-krkn-hub-tests-414

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-4.14-nightly-krkn-hub-tests-414

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-4.14-nightly-krkn-hub-tests-414

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-4.14-nightly-krkn-hub-tests-414

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-4.14-nightly-krkn-hub-tests-414

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-aro-4.13-nightly-krkn-hub-tests-aro-413

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-4.14-nightly-krkn-hub-tests-414

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-4.14-nightly-krkn-hub-tests-414

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-4.14-nightly-krkn-hub-tests-414

**Comment by paigerube14** [comment]
@chaitanyaenr @shahsahil264 can you ptal

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-4.14-nightly-krkn-hub-tests-414

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-4.14-nightly-krkn-hub-tests-414

**Comment by paigerube14** [comment]
`We aware there is an issue impacting the performance of build08. SRE identified as a known issue which they are solving how.`

**Comment by paigerube14** [comment]
@abhat @danwinship @dcbw @jacobtanenbaum the failure on this job is coming from ovn-kuberenetes jobs. Can I run make updates and add the files and you give approval or do you want to do in your own PR? 

changes are these: https://github.com/openshift/release/pull/39697/commits/6e7b9e980969856a905e9a4c47a767101fa8285f

`diff -Naupr -I '^[[:space:]]*#.*' /tmp/tmp.KFnOOeg5Co/jobs/openshift/ovn-kubernetes/openshift-ovn-kubernetes-release-4.12-presubmits.yaml .//ci-operator/jobs/openshift/ovn-kubernetes/openshift-ovn-kubernetes-release-4.12-presubmits.yaml`

**Comment by paigerube14** [comment]
@chaitanyaenr can you ptal, the jobs in other folders are no longer blocking this

**Review by shahsahil264 — APPROVED** [review]
/lgtm

**Pull request merged**